Dockerfile.ubuntu

```diff
@@ -1,4 +1,4 @@
-FROM golang:alpine as build
+FROM ubuntu:23.10 as build
 RUN apk --no-cache add ca-certificates
```

Comment:
ubuntu uses `apt` or `apt-get`, not `apk`

Does running `goreleaser release --snapshot --clean` compile for you?

Reply:
good call .. updated the Dockerfile and now it compiles fine.